lib/python/rosie/db_create.py

```diff
@@ -185,9 +201,21 @@ def load(self, repos_path):
                     "\r%s... loading revision %d of %d" %
                     (Reporter.PREFIX_INFO, revision, youngest))
                 sys.stdout.flush()
-            self.post_commit_hook.run(
-                repos_path, str(revision), no_notification=True)
-            event = RosieDatabaseLoadEvent(repos_path, revision, youngest)
+            try:
+                self.post_commit_hook.run(
+                    repos_path, str(revision), no_notification=True)
+            except (ConfigSyntaxError, DatabaseError, InfoFileError) as err:
+                if sys.stdout.isatty():
+                    sys.stdout.write("\r")
```

Comment:
`\r`?

Is the intention to rewrite the current line?

Reply:
I think that's for when it's writing to the terminal, so it clears the previous loading message or something like that